test/data/azure/create_basic_models.py

```diff
@@ -33,23 +33,23 @@ def create_audio_model():
     onnx.save(model, 'openai_audio.onnx')
 
 
-def create_chat_model():
+def create_azure_chat_model():
     auth_token = helper.make_tensor_value_info('auth_token', TensorProto.STRING, [-1])
     chat = helper.make_tensor_value_info('chat', TensorProto.STRING, [-1])
     response = helper.make_tensor_value_info('response', TensorProto.STRING, [-1])
 
     invoker = helper.make_node('AzureTextToText', ['auth_token', 'chat'], ['response'],
                                domain='com.microsoft.extensions',
                                name='chat_invoker',
-                               model_uri='https://api.openai.com/v1/chat/completions',
+                               model_uri='https://rashuai-aoai-test.openai.azure.com/openai/deployments/randysgpt/chat/completions?api-version=2023-05-15',
```

Comment:
is this endpoint temporary or forever?

Reply:
it temporary, only serving UT request for 1.16 release.